### PR TITLE
dbeaver/dbeaver#21399 relation btw close entities

### LINKED ISSUE #21399: ERD relation between close entities looks incorect.
### Description

Observed behaviour: 
Entities in ER diagram layouting close to each other have incorrect drawing relation.

Expected:
Provide dynamic behaviour of indentation value for contributed type of notations.

### DBeaver Version

Community Edition Version 23.2.1

### Operating System

All

### Database and driver

SQLite

### Steps to reproduce

1. Open default ERD diagram 
2. Change notation to EDF1X by right click

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by serjiokov**
Provided dynamic mechanism for handling indentation values for specific types of notation.
Fixed font size for label notation.

### CODE REVIEW

**Comment by serjiokov** [comment]
1. EDF1X - has indentation equal 5.
image

2. Crows Foot - has indentation eqaul 30.
image

Supporting  autolayout for entity relation representation.

File: plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java [hunk 1]
```diff
@@ -52,4 +52,11 @@ public void applyNotationForEntities(
         @NotNull ERDAssociation association,
         @NotNull Color bckColor,
         @NotNull Color frgColor);
+
+    /**
+     * Method designed to provide specific value of indentation
+     *
+     * @return - value of indentation
+     */
+    public double getIndentation();
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java:62]: Indentation between what?

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java:62]: Indentation value is a length of orthogonal line for source and target decorator. Each notation keeps own value to handle require behavior.

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java:62]: Can you clarify that in the method's name?

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java:62]: The suitable place for it is a java doc.

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java:62]:
```
@return - value of indentation
```

Yeah..

File: plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java [hunk 2]
```diff
@@ -52,4 +52,12 @@ public void applyNotationForEntities(
...
         @NotNull Color bckColor,
         @NotNull Color frgColor);
+
+    /**
+     * Indentation value is a length of orthogonal line for source and target
+     * decorator. Each notation keeps own value to handle require behavior.
+     *
+     * @return - value of indentation
+     */
+    public double getIndentation();
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java:62]: This javadoc makes no sense, unfortunately. The terms "identation" and "length" mean different things.

**Review by E1izabeth — APPROVED** [review]

**Comment by Matvey16** [comment]
Is it expected that now the diagram take more space and dots on the end if the relation lines are now bigger and grey instead of black?

**Comment by serjiokov** [comment]
Fixed ratation of EDF1X decorators:
image

**Comment by Matvey16** [comment]
Verified

**Pull request merged**